specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -4473,6 +4473,52 @@
             }
           }
         }
+      },
+      "put": {
+        "tags": [
+          "BillingRoleAssignments"
+        ],
+        "operationId": "EnrollmentRoleAssignments_Put",
```

Comment:
Can we remove "Enrollment" from the naming here? This is a billing account-level scope, which is also used outside of EA.